Automattic/newspack-ads
# Add Gutenberg icon and replace Material

## Changes proposed in this Pull Request:
This PR installs npm package for wordpress/icons (and update calypso) and replaces the Material icons, and also updates the Newspack logo to the latest one.

## How to test the changes in this Pull Request:
1. run npm install
2. check if the new icon is being displayed properly
